R/draws-index.R

```diff
@@ -141,6 +141,42 @@ remove_reserved_variable_names <- function(variables, reserved) {
   x
 }
 
+#' Set variable names in `draws` objects
+#'
+#' Set variable names for all variables in a [`draws`] object. Useful
+#' when using pipe operators.
+#'
+#' @param .x (draws) A [`draws`] object.
+#' @param variables (character) new variable names.
+#' @template args-methods-dots
+#'
+#' @return Returns a [`draws`] object of the same format as `.x`, with
+#'   variables named as specified.
+#'
+#' @seealso [`variables`]
+#' @examples
+#' x <- as_draws(matrix(rnorm(100), ncol = 2))
+#' variables(x)
+#'
+#' x |> set_variables(c("theta[1]", "theta[2]")) |> variables()
+#'
+#' # this is equivalent to
+#' variables(x) <- c("theta[1]", "theta[2]")
+#' variables(x)
+#'
+#' @export
+set_variables <- function(.x, ...) {
```

Comment:
why use .x as the first argument here? Wouldn't x be okay too?

Reply:
Fixed by changing variable to `x`